apps/full-site-editing/blank-theme/index.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+get_header();
+
+$template_hierarchy = json_decode( get_option( 'template_hierarchy' ) );
+
+if ( is_singular() && have_posts() ) {
+	the_post();
+
+	$post_id = get_the_ID();
+	$template_id = get_post_meta( $post_id, 'wp_template_id', true );
+
+	if ( ! $template_id ) {
+		if ( isset( $template_hierarchy->singular ) ) {
+			$template_id = $template_hierarchy->singular;
+		} else if ( isset( $template_hierarchy->index ) ) {
+			$template_id = $template_hierarchy->index;
+		}
+	}
+
+	$template = get_post( $template_id );
```

Comment:
we need something here for post types with no `$template_id`. On my clean install, I tried
```php
$template = get_post( $template_id ?? null );
```
_yeah, null-coalescing operator because PHP 7 has it and I won't listen otherwise._

| Before | After |
| - | - |
| ![before](https://user-images.githubusercontent.com/233601/57562639-da6fd300-736a-11e9-9150-bf146b85bcd1.png) | ![after](https://user-images.githubusercontent.com/233601/57562647-ee1b3980-736a-11e9-8e87-46cd4e02428b.png) |